package.json

```
@@ -52,7 +52,7 @@
     "oclif": "^3.0.1",
     "ts-jest": "^28.0.5",
     "ts-node": "^10.7.0",
-    "typescript": "^4.6.4"
+    "typescript": "4.9.3"
```

```suggestion
    "typescript": "^4.9.3"
```

Or was this intentional?

---

Right, thanks!